** ComfyUI startup time: 2024-02-27 11:54:32.848522
[2024-02-27 11:54] ** Platform: Windows
[2024-02-27 11:54] ** Python version: 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
[2024-02-27 11:54] ** Python executable: C:\Python310\python.exe
[2024-02-27 11:54] ** Log path: C:\Users\Lightmapper\Documents\GitHub\ComfyUI\comfyui.log
[2024-02-27 11:54] 
#######################################################################
[2024-02-27 11:54] [ComfyUI-Manager] Starting dependency installation/(de)activation for the extension
[2024-02-27 11:54] 
[2024-02-27 11:54] Install: pip packages for 'C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\comfyui-reactor-node'
[!] WARNING: Ignoring invalid distribution -illow (c:\users\lightmapper\appdata\roaming\python\python310\site-packages)
[2024-02-27 11:54] [!] WARNING: Ignoring invalid distribution -illow (c:\users\lightmapper\appdata\roaming\python\python310\site-packages)
[2024-02-27 11:54] [2024-02-27 11:54]  Collecting insightface==0.7.3
[2024-02-27 11:54]    Using cached insightface-0.7.3.tar.gz (439 kB)
[2024-02-27 11:54]    Installing build dependencies: started
[2024-02-27 11:54]    Installing build dependencies: finished with status 'done'
[2024-02-27 11:54]    Getting requirements to build wheel: started
[2024-02-27 11:54]    Getting requirements to build wheel: finished with status 'done'
[2024-02-27 11:54]    Installing backend dependencies: started
[2024-02-27 11:54]    Installing backend dependencies: finished with status 'done'
[2024-02-27 11:54]    Preparing metadata (pyproject.toml): started
[2024-02-27 11:54]    Preparing metadata (pyproject.toml): finished with status 'done'
[2024-02-27 11:54]           Collecting easydict (from insightface==0.7.3)
[2024-02-27 11:54]    Obtaining dependency information for easydict from https://files.pythonhosted.org/packages/3b/37/9a0c29a3c8ec80c54ff9132160bfc5c6cc6fedda073b391c2a06d9ad0499/easydict-1.12-py3-none-any.whl.metadata
[2024-02-27 11:54]    Using cached easydict-1.12-py3-none-any.whl.metadata (4.1 kB)
[2024-02-27 11:54]  Collecting cython (from insightface==0.7.3)
[2024-02-27 11:54]    Obtaining dependency information for cython from https://files.pythonhosted.org/packages/18/15/fdc0c3552d20f9337b134a36d786da24e47998fc39f62cb61c1534f26123/Cython-3.0.8-cp310-cp310-win_amd64.whl.metadata
[2024-02-27 11:54]    Using cached Cython-3.0.8-cp310-cp310-win_amd64.whl.metadata (3.2 kB)
[2024-02-27 11:54]   Collecting prettytable (from insightface==0.7.3)
[2024-02-27 11:54]    Obtaining dependency information for prettytable from https://files.pythonhosted.org/packages/3d/c4/a32f4bf44faf95accbbd5d7864ddef9e289749a8efbc3adaad4a4671779a/prettytable-3.10.0-py3-none-any.whl.metadata
[2024-02-27 11:54]    Using cached prettytable-3.10.0-py3-none-any.whl.metadata (30 kB)
[2024-02-27 11:54]                            Using cached Cython-3.0.8-cp310-cp310-win_amd64.whl (2.8 MB)
[2024-02-27 11:54]  Using cached easydict-1.12-py3-none-any.whl (6.7 kB)
[2024-02-27 11:54]  Using cached prettytable-3.10.0-py3-none-any.whl (28 kB)
[2024-02-27 11:54]  Building wheels for collected packages: insightface
[2024-02-27 11:54]    Building wheel for insightface (pyproject.toml): started
[2024-02-27 11:54]    Building wheel for insightface (pyproject.toml): finished with status 'error'
[2024-02-27 11:54] [!]   error: subprocess-exited-with-error
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]   Building wheel for insightface (pyproject.toml) did not run successfully.
[2024-02-27 11:54] [!]   exit code: 1
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]   [215 lines of output]
[2024-02-27 11:54] [!]   WARNING: pandoc not enabled
[2024-02-27 11:54] [!]   running bdist_wheel
[2024-02-27 11:54] [!]   running build
[2024-02-27 11:54] [!]   running build_py
[2024-02-27 11:54] [!]   creating build
[2024-02-27 11:54] [!]   creating build\lib.win-amd64-cpython-310
[2024-02-27 11:54] [!]   creating build\lib.win-amd64-cpython-310\insightface
[2024-02-27 11:54] [!]   copying insightface\__init__.py -> build\lib.win-amd64-cpython-310\insightface
[2024-02-27 11:54] [!]   creating build\lib.win-amd64-cpython-310\insightface\app
[2024-02-27 11:54] [!]   copying insightface\app\common.py -> build\lib.win-amd64-cpython-310\insightface\app
[2024-02-27 11:54] [!]   copying insightface\app\face_analysis.py -> build\lib.win-amd64-cpython-310\insightface\app
[2024-02-27 11:54] [!]   copying insightface\app\mask_renderer.py -> build\lib.win-amd64-cpython-310\insightface\app
[2024-02-27 11:54] [!]   copying insightface\app\__init__.py -> build\lib.win-amd64-cpython-310\insightface\app
[2024-02-27 11:54] [!]   creating build\lib.win-amd64-cpython-310\insightface\commands
[2024-02-27 11:54] [!]   copying insightface\commands\insightface_cli.py -> build\lib.win-amd64-cpython-310\insightface\commands
[2024-02-27 11:54] [!]   copying insightface\commands\model_download.py -> build\lib.win-amd64-cpython-310\insightface\commands
[2024-02-27 11:54] [!]   copying insightface\commands\rec_add_mask_param.py -> build\lib.win-amd64-cpython-310\insightface\commands
[2024-02-27 11:54] [!]   copying insightface\commands\__init__.py -> build\lib.win-amd64-cpython-310\insightface\commands
[2024-02-27 11:54] [!]   creating build\lib.win-amd64-cpython-310\insightface\data
[2024-02-27 11:54] [!]   copying insightface\data\image.py -> build\lib.win-amd64-cpython-310\insightface\data
[2024-02-27 11:54] [!]   copying insightface\data\pickle_object.py -> build\lib.win-amd64-cpython-310\insightface\data
[2024-02-27 11:54] [!]   copying insightface\data\rec_builder.py -> build\lib.win-amd64-cpython-310\insightface\data
[2024-02-27 11:54] [!]   copying insightface\data\__init__.py -> build\lib.win-amd64-cpython-310\insightface\data
[2024-02-27 11:54] [!]   creating build\lib.win-amd64-cpython-310\insightface\model_zoo
[2024-02-27 11:54] [!]   copying insightface\model_zoo\arcface_onnx.py -> build\lib.win-amd64-cpython-310\insightface\model_zoo
[2024-02-27 11:54] [!]   copying insightface\model_zoo\attribute.py -> build\lib.win-amd64-cpython-310\insightface\model_zoo
[2024-02-27 11:54] [!]   copying insightface\model_zoo\inswapper.py -> build\lib.win-amd64-cpython-310\insightface\model_zoo
[2024-02-27 11:54] [!]   copying insightface\model_zoo\landmark.py -> build\lib.win-amd64-cpython-310\insightface\model_zoo
[2024-02-27 11:54] [!]   copying insightface\model_zoo\model_store.py -> build\lib.win-amd64-cpython-310\insightface\model_zoo
[2024-02-27 11:54] [!]   copying insightface\model_zoo\model_zoo.py -> build\lib.win-amd64-cpython-310\insightface\model_zoo
[2024-02-27 11:54] [!]   copying insightface\model_zoo\retinaface.py -> build\lib.win-amd64-cpython-310\insightface\model_zoo
[2024-02-27 11:54] [!]   copying insightface\model_zoo\scrfd.py -> build\lib.win-amd64-cpython-310\insightface\model_zoo
[2024-02-27 11:54] [!]   copying insightface\model_zoo\__init__.py -> build\lib.win-amd64-cpython-310\insightface\model_zoo
[2024-02-27 11:54] [!]   creating build\lib.win-amd64-cpython-310\insightface\thirdparty
[2024-02-27 11:54] [!]   copying insightface\thirdparty\__init__.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty
[2024-02-27 11:54] [!]   creating build\lib.win-amd64-cpython-310\insightface\utils
[2024-02-27 11:54] [!]   copying insightface\utils\constant.py -> build\lib.win-amd64-cpython-310\insightface\utils
[2024-02-27 11:54] [!]   copying insightface\utils\download.py -> build\lib.win-amd64-cpython-310\insightface\utils
[2024-02-27 11:54] [!]   copying insightface\utils\face_align.py -> build\lib.win-amd64-cpython-310\insightface\utils
[2024-02-27 11:54] [!]   copying insightface\utils\filesystem.py -> build\lib.win-amd64-cpython-310\insightface\utils
[2024-02-27 11:54] [!]   copying insightface\utils\storage.py -> build\lib.win-amd64-cpython-310\insightface\utils
[2024-02-27 11:54] [!]   copying insightface\utils\transform.py -> build\lib.win-amd64-cpython-310\insightface\utils
[2024-02-27 11:54] [!]   copying insightface\utils\__init__.py -> build\lib.win-amd64-cpython-310\insightface\utils
[2024-02-27 11:54] [!]   creating build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d
[2024-02-27 11:54] [!]   copying insightface\thirdparty\face3d\__init__.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d
[2024-02-27 11:54] [!]   creating build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh
[2024-02-27 11:54] [!]   copying insightface\thirdparty\face3d\mesh\io.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh
[2024-02-27 11:54] [!]   copying insightface\thirdparty\face3d\mesh\light.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh
[2024-02-27 11:54] [!]   copying insightface\thirdparty\face3d\mesh\render.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh
[2024-02-27 11:54] [!]   copying insightface\thirdparty\face3d\mesh\transform.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh
[2024-02-27 11:54] [!]   copying insightface\thirdparty\face3d\mesh\vis.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh
[2024-02-27 11:54] [!]   copying insightface\thirdparty\face3d\mesh\__init__.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh
[2024-02-27 11:54] [!]   creating build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh_numpy
[2024-02-27 11:54] [!]   copying insightface\thirdparty\face3d\mesh_numpy\io.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh_numpy
[2024-02-27 11:54] [!]   copying insightface\thirdparty\face3d\mesh_numpy\light.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh_numpy
[2024-02-27 11:54] [!]   copying insightface\thirdparty\face3d\mesh_numpy\render.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh_numpy
[2024-02-27 11:54] [!]   copying insightface\thirdparty\face3d\mesh_numpy\transform.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh_numpy
[2024-02-27 11:54] [!]   copying insightface\thirdparty\face3d\mesh_numpy\vis.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh_numpy
[2024-02-27 11:54] [!]   copying insightface\thirdparty\face3d\mesh_numpy\__init__.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh_numpy
[2024-02-27 11:54] [!]   creating build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\morphable_model
[2024-02-27 11:54] [!]   copying insightface\thirdparty\face3d\morphable_model\fit.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\morphable_model
[2024-02-27 11:54] [!]   copying insightface\thirdparty\face3d\morphable_model\load.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\morphable_model
[2024-02-27 11:54] [!]   copying insightface\thirdparty\face3d\morphable_model\morphabel_model.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\morphable_model
[2024-02-27 11:54] [!]   copying insightface\thirdparty\face3d\morphable_model\__init__.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\morphable_model
[2024-02-27 11:54] [!]   running egg_info
[2024-02-27 11:54] [!]   writing insightface.egg-info\PKG-INFO
[2024-02-27 11:54] [!]   writing dependency_links to insightface.egg-info\dependency_links.txt
[2024-02-27 11:54] [!]   writing entry points to insightface.egg-info\entry_points.txt
[2024-02-27 11:54] [!]   writing requirements to insightface.egg-info\requires.txt
[2024-02-27 11:54] [!]   writing top-level names to insightface.egg-info\top_level.txt
[2024-02-27 11:54] [!]   reading manifest file 'insightface.egg-info\SOURCES.txt'
[2024-02-27 11:54] [!]   writing manifest file 'insightface.egg-info\SOURCES.txt'
[2024-02-27 11:54] [!]   C:\Users\Lightmapper\AppData\Local\Temp\pip-build-env-ls0mc8go\overlay\Lib\site-packages\setuptools\command\build_py.py:207: _Warning: Package 'insightface.thirdparty.face3d.mesh.cython' is absent from the `packages` configuration.
[2024-02-27 11:54] [!]   !!
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           ********************************************************************************
[2024-02-27 11:54] [!]           ############################
[2024-02-27 11:54] [!]           # Package would be ignored #
[2024-02-27 11:54] [!]           ############################
[2024-02-27 11:54] [!]           Python recognizes 'insightface.thirdparty.face3d.mesh.cython' as an importable package[^1],
[2024-02-27 11:54] [!]           but it is absent from setuptools' `packages` configuration.
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           This leads to an ambiguous overall configuration. If you want to distribute this
[2024-02-27 11:54] [!]           package, please make sure that 'insightface.thirdparty.face3d.mesh.cython' is explicitly added
[2024-02-27 11:54] [!]           to the `packages` configuration field.
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           Alternatively, you can also rely on setuptools' discovery methods
[2024-02-27 11:54] [!]           (for example by using `find_namespace_packages(...)`/`find_namespace:`
[2024-02-27 11:54] [!]           instead of `find_packages(...)`/`find:`).
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           You can read more about "package discovery" on setuptools documentation page:
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           - https://setuptools.pypa.io/en/latest/userguide/package_discovery.html
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           If you don't want 'insightface.thirdparty.face3d.mesh.cython' to be distributed and are
[2024-02-27 11:54]  [!]Failed to build insightface
 [2024-02-27 11:54]           already explicitly excluding 'insightface.thirdparty.face3d.mesh.cython' via
[2024-02-27 11:54] [!]           `find_namespace_packages(...)/find_namespace` or `find_packages(...)/find`,
[2024-02-27 11:54] [!]           you can try to use `exclude_package_data`, or `include-package-data=False` in
[2024-02-27 11:54] [!]           combination with a more fine grained `package-data` configuration.
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           You can read more about "package data files" on setuptools documentation page:
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           - https://setuptools.pypa.io/en/latest/userguide/datafiles.html
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           [^1]: For Python, any directory (with suitable naming) can be imported,
[2024-02-27 11:54] [!]                 even if it does not contain any `.py` files.
[2024-02-27 11:54] [!]                 On the other hand, currently there is no concept of package data
[2024-02-27 11:54] [!]                 directory, all directories are treated like packages.
[2024-02-27 11:54] [!]           ********************************************************************************
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]   !!
[2024-02-27 11:54] [!]     check.warn(importable)
[2024-02-27 11:54] [!]   C:\Users\Lightmapper\AppData\Local\Temp\pip-build-env-ls0mc8go\overlay\Lib\site-packages\setuptools\command\build_py.py:207: _Warning: Package 'insightface.data.images' is absent from the `packages` configuration.
[2024-02-27 11:54] [!]   !!
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           ********************************************************************************
[2024-02-27 11:54] [!]           ############################
[2024-02-27 11:54] [!]           # Package would be ignored #
[2024-02-27 11:54] [!]           ############################
[2024-02-27 11:54] [!]           Python recognizes 'insightface.data.images' as an importable package[^1],
[2024-02-27 11:54] [!]           but it is absent from setuptools' `packages` configuration.
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           This leads to an ambiguous overall configuration. If you want to distribute this
[2024-02-27 11:54] [!]           package, please make sure that 'insightface.data.images' is explicitly added
[2024-02-27 11:54] [!]           to the `packages` configuration field.
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           Alternatively, you can also rely on setuptools' discovery methods
[2024-02-27 11:54] [!]           (for example by using `find_namespace_packages(...)`/`find_namespace:`
[2024-02-27 11:54] [!]           instead of `find_packages(...)`/`find:`).
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           You can read more about "package discovery" on setuptools documentation page:
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           - https://setuptools.pypa.io/en/latest/userguide/package_discovery.html
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           If you don't want 'insightface.data.images' to be distributed and are
[2024-02-27 11:54] [!]           already explicitly excluding 'insightface.data.images' via
[2024-02-27 11:54] [!]           `find_namespace_packages(...)/find_namespace` or `find_packages(...)/find`,
[2024-02-27 11:54] [!]           you can try to use `exclude_package_data`, or `include-package-data=False` in
[2024-02-27 11:54] [!]           combination with a more fine grained `package-data` configuration.
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           You can read more about "package data files" on setuptools documentation page:
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           - https://setuptools.pypa.io/en/latest/userguide/datafiles.html
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           [^1]: For Python, any directory (with suitable naming) can be imported,
[2024-02-27 11:54] [!]                 even if it does not contain any `.py` files.
[2024-02-27 11:54] [!]                 On the other hand, currently there is no concept of package data
[2024-02-27 11:54] [!]                 directory, all directories are treated like packages.
[2024-02-27 11:54] [!]           ********************************************************************************
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]   !!
[2024-02-27 11:54] [!]     check.warn(importable)
[2024-02-27 11:54] [!]   C:\Users\Lightmapper\AppData\Local\Temp\pip-build-env-ls0mc8go\overlay\Lib\site-packages\setuptools\command\build_py.py:207: _Warning: Package 'insightface.data.objects' is absent from the `packages` configuration.
[2024-02-27 11:54] [!]   !!
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           ********************************************************************************
[2024-02-27 11:54] [!]           ############################
[2024-02-27 11:54] [!]           # Package would be ignored #
[2024-02-27 11:54] [!]           ############################
[2024-02-27 11:54] [!]           Python recognizes 'insightface.data.objects' as an importable package[^1],
[2024-02-27 11:54] [!]           but it is absent from setuptools' `packages` configuration.
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           This leads to an ambiguous overall configuration. If you want to distribute this
[2024-02-27 11:54] [!]           package, please make sure that 'insightface.data.objects' is explicitly added
[2024-02-27 11:54] [!]           to the `packages` configuration field.
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           Alternatively, you can also rely on setuptools' discovery methods
[2024-02-27 11:54] [!]           (for example by using `find_namespace_packages(...)`/`find_namespace:`
[2024-02-27 11:54] [!]           instead of `find_packages(...)`/`find:`).
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           You can read more about "package discovery" on setuptools documentation page:
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           - https://setuptools.pypa.io/en/latest/userguide/package_discovery.html
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           If you don't want 'insightface.data.objects' to be distributed and are
[2024-02-27 11:54] [!]           already explicitly excluding 'insightface.data.objects' via
[2024-02-27 11:54] [!]           `find_namespace_packages(...)/find_namespace` or `find_packages(...)/find`,
[2024-02-27 11:54] [!]           you can try to use `exclude_package_data`, or `include-package-data=False` in
[2024-02-27 11:54] [!]           combination with a more fine grained `package-data` configuration.
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           You can read more about "package data files" on setuptools documentation page:
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           - https://setuptools.pypa.io/en/latest/userguide/datafiles.html
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]           [^1]: For Python, any directory (with suitable naming) can be imported,
[2024-02-27 11:54] [!]                 even if it does not contain any `.py` files.
[2024-02-27 11:54] [!]                 On the other hand, currently there is no concept of package data
[2024-02-27 11:54] [!]                 directory, all directories are treated like packages.
[2024-02-27 11:54] [!]           ********************************************************************************
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]   !!
[2024-02-27 11:54] [!]     check.warn(importable)
[2024-02-27 11:54] [!]   creating build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh\cython
[2024-02-27 11:54] [!]   copying insightface\thirdparty\face3d\mesh\cython\mesh_core.cpp -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh\cython
[2024-02-27 11:54] [!]   copying insightface\thirdparty\face3d\mesh\cython\mesh_core.h -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh\cython
[2024-02-27 11:54] [!]   copying insightface\thirdparty\face3d\mesh\cython\mesh_core_cython.cpp -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh\cython
[2024-02-27 11:54] [!]   creating build\lib.win-amd64-cpython-310\insightface\data\images
[2024-02-27 11:54] [!]   copying insightface\data\images\Tom_Hanks_54745.png -> build\lib.win-amd64-cpython-310\insightface\data\images
[2024-02-27 11:54] [!]   copying insightface\data\images\mask_black.jpg -> build\lib.win-amd64-cpython-310\insightface\data\images
[2024-02-27 11:54] [!]   copying insightface\data\images\mask_blue.jpg -> build\lib.win-amd64-cpython-310\insightface\data\images
[2024-02-27 11:54] [!]   copying insightface\data\images\mask_green.jpg -> build\lib.win-amd64-cpython-310\insightface\data\images
[2024-02-27 11:54] [!]   copying insightface\data\images\mask_white.jpg -> build\lib.win-amd64-cpython-310\insightface\data\images
[2024-02-27 11:54] [!]   copying insightface\data\images\t1.jpg -> build\lib.win-amd64-cpython-310\insightface\data\images
[2024-02-27 11:54] [!]   creating build\lib.win-amd64-cpython-310\insightface\data\objects
[2024-02-27 11:54] [!]   copying insightface\data\objects\meanshape_68.pkl -> build\lib.win-amd64-cpython-310\insightface\data\objects
[2024-02-27 11:54] [!]   copying insightface\thirdparty\face3d\mesh\cython\mesh_core_cython.c -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh\cython
[2024-02-27 11:54] [!]   copying insightface\thirdparty\face3d\mesh\cython\mesh_core_cython.cpp -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh\cython
[2024-02-27 11:54] [!]   copying insightface\thirdparty\face3d\mesh\cython\mesh_core_cython.pyx -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh\cython
[2024-02-27 11:54] [!]   copying insightface\thirdparty\face3d\mesh\cython\setup.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh\cython
[2024-02-27 11:54] [!]   running build_ext
[2024-02-27 11:54] [!]   building 'insightface.thirdparty.face3d.mesh.cython.mesh_core_cython' extension
[2024-02-27 11:54] [!]   error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
[2024-02-27 11:54] [!]   [end of output]
[2024-02-27 11:54] [!]   
[2024-02-27 11:54] [!]   note: This error originates from a subprocess, and is likely not a problem with pip.
[2024-02-27 11:54] [!]   ERROR: Failed building wheel for insightface
[2024-02-27 11:54] [!] ERROR: Could not build wheels for insightface, which is required to install pyproject.toml-based projects
[2024-02-27 11:54] [!] WARNING: Ignoring invalid distribution -illow (c:\users\lightmapper\appdata\roaming\python\python310\site-packages)
[2024-02-27 11:54] [!] 
[2024-02-27 11:54] [!] [notice] A new release of pip is available: 23.2.1 -> 24.0
[2024-02-27 11:54] [!] [notice] To update, run: python.exe -m pip install --upgrade pip
[2024-02-27 11:54] Install: install script for 'C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\comfyui-reactor-node'
[!] WARNING: Ignoring invalid distribution -illow (c:\users\lightmapper\appdata\roaming\python\python310\site-packages)
[2024-02-27 11:54] [!] WARNING: Ignoring invalid distribution -illow (c:\users\lightmapper\appdata\roaming\python\python310\site-packages)
[2024-02-27 11:54] [2024-02-27 11:54]  Looking in indexes: https://pypi.org/simple, https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-cuda-12/pypi/simple/
[2024-02-27 11:54]           [!] WARNING: Ignoring invalid distribution -illow (c:\users\lightmapper\appdata\roaming\python\python310\site-packages)
[2024-02-27 11:54] [!] WARNING: Ignoring invalid distribution -illow (c:\users\lightmapper\appdata\roaming\python\python310\site-packages)
[2024-02-27 11:54] [!] 
[2024-02-27 11:54] [!] [notice] A new release of pip is available: 23.2.1 -> 24.0
[2024-02-27 11:54] [!] [notice] To update, run: python.exe -m pip install --upgrade pip
[2024-02-27 11:54] [!] WARNING: Ignoring invalid distribution -illow (c:\users\lightmapper\appdata\roaming\python\python310\site-packages)
[2024-02-27 11:54] [!] WARNING: Ignoring invalid distribution -illow (c:\users\lightmapper\appdata\roaming\python\python310\site-packages)
[2024-02-27 11:54]  Collecting insightface==0.7.3
[2024-02-27 11:54]    Using cached insightface-0.7.3.tar.gz (439 kB)
[2024-02-27 11:54]    Installing build dependencies: started
[2024-02-27 11:54]    Installing build dependencies: finished with status 'done'
[2024-02-27 11:54]    Getting requirements to build wheel: started
[2024-02-27 11:54]    Getting requirements to build wheel: finished with status 'done'
[2024-02-27 11:55]    Installing backend dependencies: started
[2024-02-27 11:55]    Installing backend dependencies: finished with status 'done'
[2024-02-27 11:55]    Preparing metadata (pyproject.toml): started
[2024-02-27 11:55]    Preparing metadata (pyproject.toml): finished with status 'done'
[2024-02-27 11:55]           Collecting easydict (from insightface==0.7.3)
[2024-02-27 11:55]    Obtaining dependency information for easydict from https://files.pythonhosted.org/packages/3b/37/9a0c29a3c8ec80c54ff9132160bfc5c6cc6fedda073b391c2a06d9ad0499/easydict-1.12-py3-none-any.whl.metadata
[2024-02-27 11:55]    Using cached easydict-1.12-py3-none-any.whl.metadata (4.1 kB)
[2024-02-27 11:55]  Collecting cython (from insightface==0.7.3)
[2024-02-27 11:55]    Obtaining dependency information for cython from https://files.pythonhosted.org/packages/18/15/fdc0c3552d20f9337b134a36d786da24e47998fc39f62cb61c1534f26123/Cython-3.0.8-cp310-cp310-win_amd64.whl.metadata
[2024-02-27 11:55]    Using cached Cython-3.0.8-cp310-cp310-win_amd64.whl.metadata (3.2 kB)
[2024-02-27 11:55]   Collecting prettytable (from insightface==0.7.3)
[2024-02-27 11:55]    Obtaining dependency information for prettytable from https://files.pythonhosted.org/packages/3d/c4/a32f4bf44faf95accbbd5d7864ddef9e289749a8efbc3adaad4a4671779a/prettytable-3.10.0-py3-none-any.whl.metadata
[2024-02-27 11:55]    Using cached prettytable-3.10.0-py3-none-any.whl.metadata (30 kB)
[2024-02-27 11:55]                            Using cached Cython-3.0.8-cp310-cp310-win_amd64.whl (2.8 MB)
[2024-02-27 11:55]  Using cached easydict-1.12-py3-none-any.whl (6.7 kB)
[2024-02-27 11:55]  Using cached prettytable-3.10.0-py3-none-any.whl (28 kB)
[2024-02-27 11:55]  Building wheels for collected packages: insightface
[2024-02-27 11:55]    Building wheel for insightface (pyproject.toml): started
[2024-02-27 11:55]    Building wheel for insightface (pyproject.toml): finished with status 'error'
[2024-02-27 11:55] [!]   error: subprocess-exited-with-error
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]   Building wheel for insightface (pyproject.toml) did not run successfully.
[2024-02-27 11:55] [!]   exit code: 1
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]   [215 lines of output]
[2024-02-27 11:55] [!]   WARNING: pandoc not enabled
[2024-02-27 11:55] [!]   running bdist_wheel
[2024-02-27 11:55] [!]   running build
[2024-02-27 11:55] [!]   running build_py
[2024-02-27 11:55] [!]   creating build
[2024-02-27 11:55] [!]   creating build\lib.win-amd64-cpython-310
[2024-02-27 11:55] [!]   creating build\lib.win-amd64-cpython-310\insightface
[2024-02-27 11:55] [!]   copying insightface\__init__.py -> build\lib.win-amd64-cpython-310\insightface
[2024-02-27 11:55] [!]   creating build\lib.win-amd64-cpython-310\insightface\app
[2024-02-27 11:55] [!]   copying insightface\app\common.py -> build\lib.win-amd64-cpython-310\insightface\app
[2024-02-27 11:55] [!]   copying insightface\app\face_analysis.py -> build\lib.win-amd64-cpython-310\insightface\app
[2024-02-27 11:55] [!]   copying insightface\app\mask_renderer.py -> build\lib.win-amd64-cpython-310\insightface\app
[2024-02-27 11:55] [!]   copying insightface\app\__init__.py -> build\lib.win-amd64-cpython-310\insightface\app
[2024-02-27 11:55] [!]   creating build\lib.win-amd64-cpython-310\insightface\commands
[2024-02-27 11:55] [!]   copying insightface\commands\insightface_cli.py -> build\lib.win-amd64-cpython-310\insightface\commands
[2024-02-27 11:55] [!]   copying insightface\commands\model_download.py -> build\lib.win-amd64-cpython-310\insightface\commands
[2024-02-27 11:55] [!]   copying insightface\commands\rec_add_mask_param.py -> build\lib.win-amd64-cpython-310\insightface\commands
[2024-02-27 11:55] [!]   copying insightface\commands\__init__.py -> build\lib.win-amd64-cpython-310\insightface\commands
[2024-02-27 11:55] [!]   creating build\lib.win-amd64-cpython-310\insightface\data
[2024-02-27 11:55] [!]   copying insightface\data\image.py -> build\lib.win-amd64-cpython-310\insightface\data
[2024-02-27 11:55] [!]   copying insightface\data\pickle_object.py -> build\lib.win-amd64-cpython-310\insightface\data
[2024-02-27 11:55] [!]   copying insightface\data\rec_builder.py -> build\lib.win-amd64-cpython-310\insightface\data
[2024-02-27 11:55] [!]   copying insightface\data\__init__.py -> build\lib.win-amd64-cpython-310\insightface\data
[2024-02-27 11:55] [!]   creating build\lib.win-amd64-cpython-310\insightface\model_zoo
[2024-02-27 11:55] [!]   copying insightface\model_zoo\arcface_onnx.py -> build\lib.win-amd64-cpython-310\insightface\model_zoo
[2024-02-27 11:55] [!]   copying insightface\model_zoo\attribute.py -> build\lib.win-amd64-cpython-310\insightface\model_zoo
[2024-02-27 11:55] [!]   copying insightface\model_zoo\inswapper.py -> build\lib.win-amd64-cpython-310\insightface\model_zoo
[2024-02-27 11:55] [!]   copying insightface\model_zoo\landmark.py -> build\lib.win-amd64-cpython-310\insightface\model_zoo
[2024-02-27 11:55] [!]   copying insightface\model_zoo\model_store.py -> build\lib.win-amd64-cpython-310\insightface\model_zoo
[2024-02-27 11:55] [!]   copying insightface\model_zoo\model_zoo.py -> build\lib.win-amd64-cpython-310\insightface\model_zoo
[2024-02-27 11:55] [!]   copying insightface\model_zoo\retinaface.py -> build\lib.win-amd64-cpython-310\insightface\model_zoo
[2024-02-27 11:55] [!]   copying insightface\model_zoo\scrfd.py -> build\lib.win-amd64-cpython-310\insightface\model_zoo
[2024-02-27 11:55] [!]   copying insightface\model_zoo\__init__.py -> build\lib.win-amd64-cpython-310\insightface\model_zoo
[2024-02-27 11:55] [!]   creating build\lib.win-amd64-cpython-310\insightface\thirdparty
[2024-02-27 11:55] [!]   copying insightface\thirdparty\__init__.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty
[2024-02-27 11:55] [!]   creating build\lib.win-amd64-cpython-310\insightface\utils
[2024-02-27 11:55] [!]   copying insightface\utils\constant.py -> build\lib.win-amd64-cpython-310\insightface\utils
[2024-02-27 11:55] [!]   copying insightface\utils\download.py -> build\lib.win-amd64-cpython-310\insightface\utils
[2024-02-27 11:55] [!]   copying insightface\utils\face_align.py -> build\lib.win-amd64-cpython-310\insightface\utils
[2024-02-27 11:55] [!]   copying insightface\utils\filesystem.py -> build\lib.win-amd64-cpython-310\insightface\utils
[2024-02-27 11:55] [!]   copying insightface\utils\storage.py -> build\lib.win-amd64-cpython-310\insightface\utils
[2024-02-27 11:55] [!]   copying insightface\utils\transform.py -> build\lib.win-amd64-cpython-310\insightface\utils
[2024-02-27 11:55] [!]   copying insightface\utils\__init__.py -> build\lib.win-amd64-cpython-310\insightface\utils
[2024-02-27 11:55] [!]   creating build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d
[2024-02-27 11:55] [!]   copying insightface\thirdparty\face3d\__init__.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d
[2024-02-27 11:55] [!]   creating build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh
[2024-02-27 11:55] [!]   copying insightface\thirdparty\face3d\mesh\io.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh
[2024-02-27 11:55] [!]   copying insightface\thirdparty\face3d\mesh\light.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh
[2024-02-27 11:55] [!]   copying insightface\thirdparty\face3d\mesh\render.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh
[2024-02-27 11:55] [!]   copying insightface\thirdparty\face3d\mesh\transform.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh
[2024-02-27 11:55] [!]   copying insightface\thirdparty\face3d\mesh\vis.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh
[2024-02-27 11:55] [!]   copying insightface\thirdparty\face3d\mesh\__init__.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh
[2024-02-27 11:55] [!]   creating build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh_numpy
[2024-02-27 11:55] [!]   copying insightface\thirdparty\face3d\mesh_numpy\io.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh_numpy
[2024-02-27 11:55] [!]   copying insightface\thirdparty\face3d\mesh_numpy\light.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh_numpy
[2024-02-27 11:55] [!]   copying insightface\thirdparty\face3d\mesh_numpy\render.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh_numpy
[2024-02-27 11:55] [!]   copying insightface\thirdparty\face3d\mesh_numpy\transform.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh_numpy
[2024-02-27 11:55] [!]   copying insightface\thirdparty\face3d\mesh_numpy\vis.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh_numpy
[2024-02-27 11:55] [!]   copying insightface\thirdparty\face3d\mesh_numpy\__init__.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh_numpy
[2024-02-27 11:55] [!]   creating build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\morphable_model
[2024-02-27 11:55] [!]   copying insightface\thirdparty\face3d\morphable_model\fit.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\morphable_model
[2024-02-27 11:55] [!]   copying insightface\thirdparty\face3d\morphable_model\load.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\morphable_model
[2024-02-27 11:55] [!]   copying insightface\thirdparty\face3d\morphable_model\morphabel_model.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\morphable_model
[2024-02-27 11:55] [!]   copying insightface\thirdparty\face3d\morphable_model\__init__.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\morphable_model
[2024-02-27 11:55] [!]   running egg_info
[2024-02-27 11:55] [!]   writing insightface.egg-info\PKG-INFO
[2024-02-27 11:55] [!]   writing dependency_links to insightface.egg-info\dependency_links.txt
[2024-02-27 11:55] [!]   writing entry points to insightface.egg-info\entry_points.txt
[2024-02-27 11:55] [!]   writing requirements to insightface.egg-info\requires.txt
[2024-02-27 11:55] [!]   writing top-level names to insightface.egg-info\top_level.txt
[2024-02-27 11:55] [!]   reading manifest file 'insightface.egg-info\SOURCES.txt'
[2024-02-27 11:55] [!]   writing manifest file 'insightface.egg-info\SOURCES.txt'
[2024-02-27 11:55] [!]   C:\Users\Lightmapper\AppData\Local\Temp\pip-build-env-uo5mmiej\overlay\Lib\site-packages\setuptools\command\build_py.py:207: _Warning: Package 'insightface.thirdparty.face3d.mesh.cython' is absent from the `packages` configuration.
[2024-02-27 11:55] [!]   !!
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           ********************************************************************************
[2024-02-27 11:55] [!]           ############################
[2024-02-27 11:55] [!]           # Package would be ignored #
[2024-02-27 11:55] [!]           ############################
[2024-02-27 11:55] [!]           Python recognizes 'insightface.thirdparty.face3d.mesh.cython' as an importable package[^1],
[2024-02-27 11:55] [!]           but it is absent from setuptools' `packages` configuration.
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           This leads to an ambiguous overall configuration. If you want to distribute this
[2024-02-27 11:55] [!]           package, please make sure that 'insightface.thirdparty.face3d.mesh.cython' is explicitly added
[2024-02-27 11:55] [!]           to the `packages` configuration field.
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           Alternatively, you can also rely on setuptools' discovery methods
[2024-02-27 11:55] [!]           (for example by using `find_namespace_packages(...)`/`find_namespace:`
[2024-02-27 11:55] [!]           instead of `find_packages(...)`/`find:`).
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           You can read more about "package discovery" on setuptools documentation page:
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           - https://setuptools.pypa.io/en/latest/userguide/package_discovery.html
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           If you don't want 'insightface.thirdparty.face3d.mesh.cython' to be distributed and are
[2024-02-27 11:55] [!]           already explicitly excluding 'insightface.thirdparty.face3d.mesh.cython' via
[2024-02-27 11:55] [!]           `find_namespace_packages(...)/find_namespace` or `find_packages(...)/find`,
[2024-02-27 11:55] [!]           you can try to use `exclude_package_data`, or `include-package-data=False` in
[2024-02-27 11:55] [!]           combination with a more fine grained `package-data` configuration.
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           You can read more about "package data files" on setuptools documentation page:
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           - https://setuptools.pypa.io/en/latest/userguide/datafiles.html
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           [^1]: For Python, any directory (with suitable naming) can be imported,
[2024-02-27 11:55] [!]                 even if it does not contain any `.py` files.
[2024-02-27 11:55] [!]                 On the other hand, currently there is no concept of package data
[2024-02-27 11:55] [!]                 directory, all directories are treated like packages.
[2024-02-27 11:55] [!]           ********************************************************************************
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]   !!
[2024-02-27 11:55] [!]     check.warn(importable)
[2024-02-27 11:55] [!]   C:\Users\Lightmapper\AppData\Local\Temp\pip-build-env-uo5mmiej\overlay\Lib\site-packages\setuptools\command\build_py.py:207: _Warning: Package 'insightface.data.images' is absent from the `packages` configuration.
[2024-02-27 11:55] [!]   !!
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           ********************************************************************************
[2024-02-27 11:55] [!]           ############################
[2024-02-27 11:55] [!]           # Package would be ignored #
[2024-02-27 11:55] [!]           ############################
[2024-02-27 11:55] [!]           Python recognizes 'insightface.data.images' as an importable package[^1],
[2024-02-27 11:55] [!]           but it is absent from setuptools' `packages` configuration.
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           This leads to an ambiguous overall configuration. If you want to distribute this
[2024-02-27 11:55] [!]           package, please make sure that 'insightface.data.images' is explicitly added
[2024-02-27 11:55] [!]           to the `packages` configuration field.
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           Alternatively, you can also rely on setuptools' discovery methods
[2024-02-27 11:55] [!]           (for example by using `find_namespace_packages(...)`/`find_namespace:`
[2024-02-27 11:55] [!]           instead of `find_packages(...)`/`find:`).
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           You can read more about "package discovery" on setuptools documentation page:
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           - https://setuptools.pypa.io/en/latest/userguide/package_discovery.html
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           If you don't want 'insightface.data.images' to be distributed and are
[2024-02-27 11:55] [!]           already explicitly excluding 'insightface.data.images' via
[2024-02-27 11:55] [!]           `find_namespace_packages(...)/find_namespace` or `find_packages(...)/find`,
[2024-02-27 11:55] [!]           you can try to use `exclude_package_data`, or `include-package-data=False` in
[2024-02-27 11:55] [!]           combination with a more fine grained `package-data` configuration.
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           You can read more about "package data files" on setuptools documentation page:
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           - https://setuptools.pypa.io/en/latest/userguide/datafiles.html
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           [^1]: For Python, any directory (with suitable naming) can be imported,
[2024-02-27 11:55] [!]                 even if it does not contain any `.py` files.
[2024-02-27 11:55] [!]                 On the other hand, currently there is no concept of package data
[2024-02-27 11:55] [!]                 directory, all directories are treated like packages.
[2024-02-27 11:55] [!]           ********************************************************************************
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]   !!
[2024-02-27 11:55] [!]     check.warn(importable)
[2024-02-27 11:55] [!]   C:\Users\Lightmapper\AppData\Local\Temp\pip-build-env-uo5mmiej\overlay\Lib\site-packages\setuptools\command\build_py.py:207: _Warning: Package 'insightface.data.objects' is absent from the `packages` configuration.
[2024-02-27 11:55] [!]   !!
[2024-02-27 11:55] [!]   
[2024-02-27 11:55]  [!]Failed to build insightface
 [2024-02-27 11:55]           ********************************************************************************
[2024-02-27 11:55] [!]           ############################
[2024-02-27 11:55] [!]           # Package would be ignored #
[2024-02-27 11:55] [!]           ############################
[2024-02-27 11:55] [!]           Python recognizes 'insightface.data.objects' as an importable package[^1],
[2024-02-27 11:55] [!]           but it is absent from setuptools' `packages` configuration.
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           This leads to an ambiguous overall configuration. If you want to distribute this
[2024-02-27 11:55] [!]           package, please make sure that 'insightface.data.objects' is explicitly added
[2024-02-27 11:55] [!]           to the `packages` configuration field.
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           Alternatively, you can also rely on setuptools' discovery methods
[2024-02-27 11:55] [!]           (for example by using `find_namespace_packages(...)`/`find_namespace:`
[2024-02-27 11:55] [!]           instead of `find_packages(...)`/`find:`).
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           You can read more about "package discovery" on setuptools documentation page:
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           - https://setuptools.pypa.io/en/latest/userguide/package_discovery.html
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           If you don't want 'insightface.data.objects' to be distributed and are
[2024-02-27 11:55] [!]           already explicitly excluding 'insightface.data.objects' via
[2024-02-27 11:55] [!]           `find_namespace_packages(...)/find_namespace` or `find_packages(...)/find`,
[2024-02-27 11:55] [!]           you can try to use `exclude_package_data`, or `include-package-data=False` in
[2024-02-27 11:55] [!]           combination with a more fine grained `package-data` configuration.
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           You can read more about "package data files" on setuptools documentation page:
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           - https://setuptools.pypa.io/en/latest/userguide/datafiles.html
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]           [^1]: For Python, any directory (with suitable naming) can be imported,
[2024-02-27 11:55] [!]                 even if it does not contain any `.py` files.
[2024-02-27 11:55] [!]                 On the other hand, currently there is no concept of package data
[2024-02-27 11:55] [!]                 directory, all directories are treated like packages.
[2024-02-27 11:55] [!]           ********************************************************************************
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]   !!
[2024-02-27 11:55] [!]     check.warn(importable)
[2024-02-27 11:55] [!]   creating build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh\cython
[2024-02-27 11:55] [!]   copying insightface\thirdparty\face3d\mesh\cython\mesh_core.cpp -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh\cython
[2024-02-27 11:55] [!]   copying insightface\thirdparty\face3d\mesh\cython\mesh_core.h -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh\cython
[2024-02-27 11:55] [!]   copying insightface\thirdparty\face3d\mesh\cython\mesh_core_cython.cpp -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh\cython
[2024-02-27 11:55] [!]   creating build\lib.win-amd64-cpython-310\insightface\data\images
[2024-02-27 11:55] [!]   copying insightface\data\images\Tom_Hanks_54745.png -> build\lib.win-amd64-cpython-310\insightface\data\images
[2024-02-27 11:55] [!]   copying insightface\data\images\mask_black.jpg -> build\lib.win-amd64-cpython-310\insightface\data\images
[2024-02-27 11:55] [!]   copying insightface\data\images\mask_blue.jpg -> build\lib.win-amd64-cpython-310\insightface\data\images
[2024-02-27 11:55] [!]   copying insightface\data\images\mask_green.jpg -> build\lib.win-amd64-cpython-310\insightface\data\images
[2024-02-27 11:55] [!]   copying insightface\data\images\mask_white.jpg -> build\lib.win-amd64-cpython-310\insightface\data\images
[2024-02-27 11:55] [!]   copying insightface\data\images\t1.jpg -> build\lib.win-amd64-cpython-310\insightface\data\images
[2024-02-27 11:55] [!]   creating build\lib.win-amd64-cpython-310\insightface\data\objects
[2024-02-27 11:55] [!]   copying insightface\data\objects\meanshape_68.pkl -> build\lib.win-amd64-cpython-310\insightface\data\objects
[2024-02-27 11:55] [!]   copying insightface\thirdparty\face3d\mesh\cython\mesh_core_cython.c -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh\cython
[2024-02-27 11:55] [!]   copying insightface\thirdparty\face3d\mesh\cython\mesh_core_cython.cpp -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh\cython
[2024-02-27 11:55] [!]   copying insightface\thirdparty\face3d\mesh\cython\mesh_core_cython.pyx -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh\cython
[2024-02-27 11:55] [!]   copying insightface\thirdparty\face3d\mesh\cython\setup.py -> build\lib.win-amd64-cpython-310\insightface\thirdparty\face3d\mesh\cython
[2024-02-27 11:55] [!]   running build_ext
[2024-02-27 11:55] [!]   building 'insightface.thirdparty.face3d.mesh.cython.mesh_core_cython' extension
[2024-02-27 11:55] [!]   error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
[2024-02-27 11:55] [!]   [end of output]
[2024-02-27 11:55] [!]   
[2024-02-27 11:55] [!]   note: This error originates from a subprocess, and is likely not a problem with pip.
[2024-02-27 11:55] [!]   ERROR: Failed building wheel for insightface
[2024-02-27 11:55] [!] ERROR: Could not build wheels for insightface, which is required to install pyproject.toml-based projects
[2024-02-27 11:55] [!] WARNING: Ignoring invalid distribution -illow (c:\users\lightmapper\appdata\roaming\python\python310\site-packages)
[2024-02-27 11:55] [!] 
[2024-02-27 11:55] [!] [notice] A new release of pip is available: 23.2.1 -> 24.0
[2024-02-27 11:55] [!] [notice] To update, run: python.exe -m pip install --upgrade pip
[2024-02-27 11:55]  CUDA 12.1
[2024-02-27 11:55]  Status: The 'insightface==0.7.3' distribution was not found and is required by the application
[2024-02-27 11:55]  Ok
[2024-02-27 11:55] 
[ComfyUI-Manager] Startup script completed.
[2024-02-27 11:55] #######################################################################
[2024-02-27 11:55] 
[2024-02-27 11:55] 
Prestartup times for custom nodes:
[2024-02-27 11:55]    0.0 seconds: C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\rgthree-comfy
[2024-02-27 11:55]   32.6 seconds: C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\ComfyUI-Manager
[2024-02-27 11:55] 
[2024-02-27 11:55] Total VRAM 24576 MB, total RAM 65451 MB
[2024-02-27 11:55] Set vram state to: NORMAL_VRAM
[2024-02-27 11:55] Device: cuda:0 NVIDIA GeForce RTX 3090 : cudaMallocAsync
[2024-02-27 11:55] VAE dtype: torch.bfloat16
[2024-02-27 11:55] Using pytorch cross attention
[2024-02-27 11:55] ### Loading: ComfyUI-Manager (V2.8.3)
[2024-02-27 11:55] 

## [WARN] ComfyUI-Manager: Your ComfyUI version (1892)[2024-01-09] is too old. Please update to the latest version. ##

[2024-02-27 11:55] 
[2024-02-27 11:55] ### ComfyUI Revision: 1892 [b3ef363f] | Released on '2024-01-09'
[2024-02-27 11:55] [0;33m[ReActor][0m - [38;5;173mSTATUS[0m - [0;32mRunning v0.4.1-b11 in ComfyUI[0m
[2024-02-27 11:55] Traceback (most recent call last):
  File "C:\Users\Lightmapper\Documents\GitHub\ComfyUI\nodes.py", line 1814, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\comfyui-reactor-node\__init__.py", line 23, in <module>
    from .nodes import NODE_CLASS_MAPPINGS, NODE_DISPLAY_NAME_MAPPINGS
  File "C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\comfyui-reactor-node\nodes.py", line 15, in <module>
    from scripts.reactor_faceswap import (
  File "C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\comfyui-reactor-node\scripts\reactor_faceswap.py", line 13, in <module>
    from scripts.reactor_logger import logger
  File "C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\comfyui-reactor-node\scripts\reactor_logger.py", line 6, in <module>
    from reactor_utils import addLoggingLevel
  File "C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\comfyui-reactor-node\reactor_utils.py", line 8, in <module>
    from insightface.app.common import Face
ModuleNotFoundError: No module named 'insightface'
[2024-02-27 11:55] 
[2024-02-27 11:55] Cannot import C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\comfyui-reactor-node module for custom nodes: No module named 'insightface'
[36;20m[comfyui_controlnet_aux] | INFO -> Using ckpts path: C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\comfyui_controlnet_aux\ckpts[0m
[2024-02-27 11:55] [36;20m[comfyui_controlnet_aux] | INFO -> Using symlinks: False[0m
[2024-02-27 11:55] [36;20m[comfyui_controlnet_aux] | INFO -> Using ort providers: ['CUDAExecutionProvider', 'DirectMLExecutionProvider', 'OpenVINOExecutionProvider', 'ROCMExecutionProvider', 'CPUExecutionProvider', 'CoreMLExecutionProvider'][0m
[2024-02-27 11:55] [ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/model-list.json
[2024-02-27 11:55] [ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/alter-list.json
[2024-02-27 11:55] [ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/extension-node-map.json
[2024-02-27 11:55] [ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/custom-node-list.json
[2024-02-27 11:55] Traceback (most recent call last):
  File "C:\Users\Lightmapper\Documents\GitHub\ComfyUI\nodes.py", line 1814, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\ComfyUI_UltimateSDUpscale\__init__.py", line 24, in <module>
    from .nodes import NODE_CLASS_MAPPINGS, NODE_DISPLAY_NAME_MAPPINGS
  File "C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\ComfyUI_UltimateSDUpscale\nodes.py", line 5, in <module>
    from usdu_patch import usdu
  File "C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\ComfyUI_UltimateSDUpscale\usdu_patch.py", line 2, in <module>
    from repositories import ultimate_upscale as usdu
  File "C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\ComfyUI_UltimateSDUpscale\repositories\__init__.py", line 14, in <module>
    spec.loader.exec_module(ultimate_upscale)
  File "C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\ComfyUI_UltimateSDUpscale\repositories\ultimate_sd_upscale\scripts\ultimate-upscale.py", line 4, in <module>
    from modules import processing, shared, images, devices, scripts
ImportError: cannot import name 'devices' from 'modules' (C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\comfyui-reactor-node\modules\__init__.py)
[2024-02-27 11:55] 
[2024-02-27 11:55] Cannot import C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\ComfyUI_UltimateSDUpscale module for custom nodes: cannot import name 'devices' from 'modules' (C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\comfyui-reactor-node\modules\__init__.py)
[2024-02-27 11:55] 
[2024-02-27 11:55] [92m[rgthree] Loaded 35 extraordinary nodes.[0m
[2024-02-27 11:55] [92m[rgthree] Will use rgthree's optimized recursive execution.[0m
[2024-02-27 11:55] 
[2024-02-27 11:55] 
Import times for custom nodes:
[2024-02-27 11:55]    0.0 seconds: C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\sdxl-recommended-res-calc
[2024-02-27 11:55]    0.0 seconds: C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\tiled_ksampler
[2024-02-27 11:55]    0.0 seconds (IMPORT FAILED): C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\ComfyUI_UltimateSDUpscale
[2024-02-27 11:55]    0.0 seconds: C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\rgthree-comfy
[2024-02-27 11:55]    0.0 seconds (IMPORT FAILED): C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\comfyui-reactor-node
[2024-02-27 11:55]    0.3 seconds: C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\ComfyUI-Manager
[2024-02-27 11:55]    0.9 seconds: C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\comfyui_controlnet_aux
[2024-02-27 11:55] 
[2024-02-27 11:55] Starting server
[2024-02-27 11:55] 
[2024-02-27 11:55] To see the GUI go to: http://127.0.0.1:8188
[2024-02-27 11:55] FETCH DATA from: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/custom-node-list.json
[2024-02-27 11:55] FETCH DATA from: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/extension-node-map.json
[2024-02-27 11:55] Update custom node 'UltimateSDUpscale'
[2024-02-27 11:55] Update: ['https://github.com/ssitu/ComfyUI_UltimateSDUpscale']
[2024-02-27 11:55] Update was successful.
[2024-02-27 11:55] After restarting ComfyUI, please refresh the browser.
[2024-02-27 11:55] FETCH DATA from: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/custom-node-list.json
[2024-02-27 11:55] FETCH DATA from: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/extension-node-map.json
[2024-02-27 11:55] Update custom node 'UltimateSDUpscale'
[2024-02-27 11:55] Update: ['https://github.com/ssitu/ComfyUI_UltimateSDUpscale']
[2024-02-27 11:56] Update was successful.
[2024-02-27 11:56] After restarting ComfyUI, please refresh the browser.
[2024-02-27 11:56] FETCH DATA from: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/custom-node-list.json
[2024-02-27 11:56] FETCH DATA from: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/extension-node-map.json
[2024-02-27 11:56] Install custom node 'UltimateSDUpscale'
[2024-02-27 11:56] Try fixing: ['https://github.com/ssitu/ComfyUI_UltimateSDUpscale']
[2024-02-27 11:56] Attempt to fixing '['https://github.com/ssitu/ComfyUI_UltimateSDUpscale']' is done.
[2024-02-27 11:56] After restarting ComfyUI, please refresh the browser.
[2024-02-27 11:56] FETCH DATA from: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/custom-node-list.json
[2024-02-27 11:56] FETCH DATA from: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/extension-node-map.json
[2024-02-27 11:56] Uninstall custom node 'UltimateSDUpscale'
[2024-02-27 11:56] uninstall: ['https://github.com/ssitu/ComfyUI_UltimateSDUpscale']
[2024-02-27 11:56]  Versteckte Datei wird nicht zurückgesetzt - C:\Users\Lightmapper\Documents\GitHub\ComfyUI\custom_nodes\ComfyUI_UltimateSDUpscale\repositories\ultimate_sd_upscale\.git
[2024-02-27 11:56] Uninstallation was successful.
[2024-02-27 11:56] After restarting ComfyUI, please refresh the browser.
[2024-02-27 11:56] FETCH DATA from: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/custom-node-list.json
[2024-02-27 11:56] FETCH DATA from: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/extension-node-map.json
